Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -0,0 +1,391 @@
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>security.deviceClipboardSharing</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Allow Universal Clipboard</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Show app and website icons</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>privacy.downloadRichIcons</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Show app and website icons</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Check for vulnerable passwords</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>privacy.checkHibp</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Check for vulnerable passwords</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Automatically check for updates</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>updates.autoUpdate</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Automatically check for updates</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
+			<key>pfm_description</key>
+			<string>Set release channel</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.1password.com/mobile-device-management/</string>
+			<key>pfm_name</key>
+			<string>updates.updateChannel</string>
+			<key>pfm_platforms</key>
+			<array>
+				<string>macOS</string>
+			</array>
+			<key>pfm_range_list</key>
+			<array>
+				<string>PRODUCTION</string>
+				<string>BETA</string>
+				<string>NIGHTLY</string>
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>Production</string>
+				<string>Beta</string>
+				<string>Nightly</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Set release channel</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+	</array>
+	<key>pfm_targets</key>
+	<array>
+		<string>system</string>
+	</array>
+	<key>pfm_title</key>
+	<string>1Password 8</string>
```

Comment:
Thinking a bit longer term, since the preference domain has been standardized to not be version specific, perhaps this shouldn't be either.

Reply:
Fix was to [update 1Password 7 manifest](https://github.com/ProfileCreator/ProfileManifests/pull/577/commits/3e56ce26a0e81a6d54a6bd2dc6540250952ffc40)